### ISSUE TITLE
Requester Terminal Support

### ISSUE THREAD

**Issue by heisluft** [Mari023/AE2WirelessTerminalLibrary#269]
As in the title, ExtendedAE adds a Requester Terminal, I'd really like to see its Wireless Version

**Comment by Mari023** [comment]
do you mean [merequester](https://modrinth.com/mod/merequester)'s Requester Terminal?
to my knowledge, ExtendedAE doesn't add a Requester Terminal.

---
a wireless requester terminal for merequester has been worked on in the past, however it was put on hold due to time constraints. (see https://github.com/AlmostReliable/merequester/pull/11)
it is likely to be worked on again in the near future, tho there are also some changes to ae2wtlib I want to make which should happen before that

**Comment by Mari023** [comment]
It is now part of merequester
